### PR TITLE
AW - Bees - Add Simon and Keven

### PR DESCRIPTION
**Pull request by LarryMatte**
screen shot 2016-08-02 at 10 15 18 am

screen shot 2016-08-02 at 10 15 25 am

screen shot 2016-08-02 at 10 15 30 am

screen shot 2016-08-02 at 10 15 46 am

screen shot 2016-08-02 at 10 15 51 am

### CODE REVIEW

File: src/main/java/com/arcbees/website/client/application/bees/bee/keven/KevenModule.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2015 ArcBees Inc.
```
Comment (olafleur) [inline src/main/java/com/arcbees/website/client/application/bees/bee/keven/KevenModule.java]: 2016

File: src/main/java/com/arcbees/website/client/application/bees/bee/keven/KevenPresenter.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2015 ArcBees Inc.
```
Comment (olafleur) [inline src/main/java/com/arcbees/website/client/application/bees/bee/keven/KevenPresenter.java]: idem

**Comment by jasonlemay** [comment]
LGTM

File: src/main/java/com/arcbees/website/shared/NameTokens.java [hunk 3]
```diff
@@ -81,6 +81,10 @@
     public static final String BEE_VINCENT_FR = BEES_FR + "/vincent-beaudoin";
     public static final String BEE_CHARLES = BEES + "/charles-antoine-richard";
     public static final String BEE_CHARLES_FR = BEES_FR + "/charles-antoine-richard";
+    public static final String BEE_KEVEN = BEES_FR + "/keven-beaulieu";
```
Comment (olafleur) [inline src/main/java/com/arcbees/website/shared/NameTokens.java]: BEES (pas BEES_FR)

**Comment by olafleur** [comment]
LGTM

**Pull request merged**